packs/equipment/drake-rifle-acid.json

```
@@ -52,14 +52,48 @@
             "value": "1"
         },
         "rules": [
+            {
+                "alternate": true,
+                "key": "CriticalSpecialization",
+                "predicate": [
+                    "item:id:{item|_id}"
+                ],
+                "text": "PF2E.SpecificRule.Equipment.DrakeRifle.CriticalSpecialization"
+            },
             {
                 "key": "Note",
                 "outcome": [
+                    "success",
                     "criticalSuccess"
                 ],
                 "selector": "{item|_id}-damage",
-                "text": "On a critical hit, the spittle clings to the target and they take persistent damage of the same type as the weapon equal to @Damage[(1d4 + @item.system.damage.dice)[persistent,acid]]{1d4 + the number of weapon damage dice}. A drake rifle does not add critical specialization effects.",
-                "title": "{item|name}"
+                "text": "PF2E.SpecificRule.Equipment.DrakeRifle.Note.Acid",
+                "title": "PF2E.Check.Result.Degree.Check.success"
+            },
+            {
+                "category": "persistent",
+                "critical": true,
+                "damageType": "acid",
+                "diceNumber": 1,
+                "dieSize": "d4",
+                "key": "DamageDice",
+                "label": "PF2E.ConditionTypePersistent",
+                "predicate": [
+                    "item:id:{item|_id}"
+                ],
+                "selector": "strike-damage"
+            },
```

```suggestion
                "selector": "{item|id}-damage"
            },
```
We can use the selector to narrow it down, so other strikes don't see the damage as an option to toggle on. I also don't think we need the label on it, we don't normally add specific labels like that to things.

---

Ah, the selector was a good find, thanks for that. 

I included the label because it would neatly merge the modifier tags in the chat, relating to the persistent damage dice and modifier.

Otherwise, you will get two modifier tags in the chat message that say the following:

> - Drake Rifle (Cold) +1d4 Persistent Cold
> - Drake Rifle (Cold) +1

Instead of:

> - 1d4 + 1 Persistent Cold

Which just didn't sit right with me, plus it doesn't convey what the +1 relates to in the former example.